NEWS.md

```diff
@@ -3,6 +3,8 @@ ClimaTimeSteppers.jl Release Notes
 
 Main
 -------
+v0.9.0
+- ![][badge-💥breaking] If saveat is a number, then it does not automatically expand to `tspan[1]:saveat:tspan[2]`.
```

Comment:
Since addressing this breaking change is very easy, can you add a line where you tell people what change they should make?